AssetEditor.sln

```diff
@@ -48,77 +48,203 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ContentProject", "ContentPr
 		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Debug|x64.Build.0 = Debug|Any CPU
+		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Debug|x86.Build.0 = Debug|Any CPU
 		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Release|Any CPU.Build.0 = Release|Any CPU
+		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Release|x64.ActiveCfg = Release|Any CPU
+		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Release|x64.Build.0 = Release|Any CPU
+		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Release|x86.ActiveCfg = Release|Any CPU
+		{07D79931-CE1A-4092-AA6F-634282D31DC8}.Release|x86.Build.0 = Release|Any CPU
 		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Debug|x64.Build.0 = Debug|Any CPU
+		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Debug|x86.Build.0 = Debug|Any CPU
 		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Release|Any CPU.Build.0 = Release|Any CPU
+		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Release|x64.ActiveCfg = Release|Any CPU
+		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Release|x64.Build.0 = Release|Any CPU
+		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Release|x86.ActiveCfg = Release|Any CPU
+		{D29AB45B-3CFD-4994-BD54-CE301704D6CA}.Release|x86.Build.0 = Release|Any CPU
+		{15AAE76D-0B83-45A2-B8AB-FD7BF38BDDC3}.Debug|Any CPU.ActiveCfg = Debug|x64
+		{15AAE76D-0B83-45A2-B8AB-FD7BF38BDDC3}.Debug|x64.ActiveCfg = Debug|x64
+		{15AAE76D-0B83-45A2-B8AB-FD7BF38BDDC3}.Debug|x86.ActiveCfg = Debug|Win32
+		{15AAE76D-0B83-45A2-B8AB-FD7BF38BDDC3}.Release|Any CPU.ActiveCfg = Release|x64
+		{15AAE76D-0B83-45A2-B8AB-FD7BF38BDDC3}.Release|x64.ActiveCfg = Release|x64
+		{15AAE76D-0B83-45A2-B8AB-FD7BF38BDDC3}.Release|x86.ActiveCfg = Release|Win32
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(NestedProjects) = preSolution
+		{FB2E6959-32F9-4F01-848B-21B38BDE98CC} = {18424B6A-CB8A-4CE1-935C-72459F31521B}
 		{98E09611-7C80-4413-9510-DE52EF71D7F1} = {18424B6A-CB8A-4CE1-935C-72459F31521B}
 		{3DC3A5F3-182F-46D4-BB7E-6639FC965E03} = {18424B6A-CB8A-4CE1-935C-72459F31521B}
 		{7FD31ACD-8D46-43FE-9E17-27198AED902B} = {C174C249-D109-49BE-8BC5-42A0D5DEDBDA}
 		{D7225D7F-87E4-4FAD-A3BE-9165DFDC315E} = {07AC615B-A8FC-4E1A-BDD5-BC11452429A0}
+		{ECAC322F-8FA4-4F98-98D5-5E18464025C7} = {FF70C4FA-A9A7-4DEC-87A3-4BF5F23EF336}
```

Comment:
Not sure which line it is, but you have moved the main assetEdior project into a folder where it should not be